recipes/pytorch-dev-deps/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: pytorch-dev-deps
+  version: 1.0
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - install -Dm664 {{ RECIPE_DIR }}/activate.sh {{ PREFIX }}/etc/conda/activate.d/activate-pytorch-dev-deps.sh  # [unix]
+
+requirements:
+  run:
+    - python >=3.6
+    - numpy !=1.20.0
+    - ninja
+    - pyyaml
+    - mkl
+    - mkl-include
+    - setuptools
+    - cmake
+    - cffi
+    - typing
+    - pytest
+    - compilers
+    - flake8
+    - psutil
+    - hypothesis
+    - mypy ==0.812
+
+test:
+  commands:
+    - echo "Environment was successfully solved!"
+
+about:
+  summary: Development dependencies to work on PyTorch.
+  home: https://github.com/pytorch/pytorch
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
What license do we use? Currently the file is just a dummy to make CI happy.

Reply:
BSD-3 sounds good to me.